### PR TITLE
New guides and concepts

### PR DESCRIPTION
**Pull request by mikesir87**
DO NOT MERGE at this time!

### Proposed changes

This PR contains all of the new guides and concepts being worked on. This PR should remain in draft until we are ready to actually deploy the new material. Having this PR enables the Netlify Preview, making it possible to do usability testing on the new content.

(The only commit here was enough to cause a change on the branch to open the PR)

### CODE REVIEW

**Comment by mikesir87** [comment]
Just rebased against latest main and resolved a conflict with `package.json` and `package-lock.json`

File: content/guides/get-started/develop-with-containers.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+With this environment up and running, let's make a simple change to the application and see how Docker helps provide a fast feedback loop.
+
+### Change the greeting
+
+The greeting at the top of the page is populated by an API call at `/api/greeting`. Currently, it always returns "Hello world!". Let's modify it to return one of three randomized messages (that you'll get to choose).
+
+1. Open the `backend/src/routes/getGreeting.js` file. This file provides the handler for the API endpoint.
+
+2. Modify the variable at the top to an array of greetings. Feel free to use the ones provided below or any customize it to your own liking!
```
Comment (craig-osterhout) [inline content/guides/get-started/develop-with-containers.md:81]:
```suggestion
2. Modify the variable at the top to an array of greetings. Feel free to use the following modifications or customize it to your own liking!
```

File: content/guides/get-started/develop-with-containers.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    ```
+
+4. If you haven't done so yet, save the file. If you refresh your browser, you should see a new greeting. If you keep refreshing, you should see all of the messages appear.
+
+    ![Screenshot of the to-do app with a new greeting of "Whalecome!"](images/develop-app-with-greetings.webp)
+
+
+### Change the placeholder text
+
+When you look at the app, you'll see the placeholder text is simply "New Item". Let's make that a little more descriptive and fun! We'll also make a few changes to the styling of the app too.
```
Comment (craig-osterhout) [inline content/guides/get-started/develop-with-containers.md:111]:
```suggestion
When you look at the app, you'll see the placeholder text is simply "New Item". You'll now make that a little more descriptive and fun! You'll also make a few changes to the styling of the app too.
```

File: content/guides/get-started/develop-with-containers.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    ```
+
+3. Save the file and go back to your browser. You should see the change already hot-reloaded into your browser. If you don't like it, feel free to tweak it until it looks just right!
+
+![Screenshot of the to-do app with an updated placeholder in the add item text field"](images/develop-app-with-updated-placeholder.webp)
+
+
+### Change the background color
+
+Before we consider the application finalized, we need to make the colors better.
```
Comment (craig-osterhout) [inline content/guides/get-started/develop-with-containers.md:134]:
```suggestion
Before you consider the application finalized, you need to make the colors better.
```

File: content/guides/get-started/develop-with-containers.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+
+    ![Screenshot of the to-do app with a new placeholder and background color"](images/develop-app-with-updated-client.webp)
+
+And with that, you're done! Congrats on updating your website! :tada:
+
+
+## Recap
+
+Before we move on, let's take a moment and reflect on what happened here. Within a few moments, you were able to:
```
Comment (craig-osterhout) [inline content/guides/get-started/develop-with-containers.md:160]:
```suggestion
Before you move on, take a moment and reflect on what happened here. Within a few moments, you were able to:
```

File: content/guides/get-started/develop-with-containers.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+- **Start a complete development project with _zero_ installation effort.** The containerized environment provided the development environment, ensuring you have everything you need. You didn't have to install Node, MySQL, or any of the other dependencies directly on your machine. All you needed was Docker Desktop and a code editor.
```
Comment (craig-osterhout) [inline content/guides/get-started/develop-with-containers.md:162]: Suggest avoiding emphasis (bold & italics)

File: content/guides/get-started/develop-with-containers.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+- **Make changes and see them immediately.** This was made possible because 1) the processes running in each container are watching and responding to file changes and 2) the files are shared with the containerized environment.
```
Comment (craig-osterhout) [inline content/guides/get-started/develop-with-containers.md:164]: Suggest avoiding emphasis (bold)

File: content/guides/get-started/get-docker-desktop.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (craig-osterhout) [inline content/guides/get-started/get-docker-desktop.md:1]: Noting that this page was moved to manuals recently. https://github.com/docker/docs/commit/951147d39fc886a7a0673db63ae83d0e281e2a3d
I see that it is more incorporated to the guide here, but it now exists in multiple places.

**Review by craig-osterhout — COMMENTED** [review]
It's coming along nicely! Added some style suggestions. Some common ones:
- Try to avoid "we/our/let's".
- Try to avoid emphasis (bold & italics).
- Try to avoid "click" in favor of "select".

**Comment by mikesir87** [comment]
Closing this PR. Will reopen when we're ready for new content.

**Pull request closed without merging**